sites/funcampus/src/frontend/scss/settings/_variables.scss

```
@@ -158,6 +158,7 @@ $r-footer-logo-width-lg: 12.125rem;
 // full width like common blocks. Usefull if you plan to remove course run from
 // template
 $r-course-aside: null;
+$r-course-subheader-aside: null;
```

Nope this value should not be null.
```suggestion
$r-course-subheader-aside: 25%;
```